docs/Contributors_Guide/dev_details/index.rst

```diff
@@ -1,6 +1,6 @@
-*******************
+###################
```

Comment:
```suggestion
*******************
```

Reply:
Development Details is at the same level as the chapter Development Environment and should have the same header.
https://met.readthedocs.io/en/feature_2716_headers/Contributors_Guide/index.html